app.js

```diff
@@ -0,0 +1,109 @@
+let board = ['', '', '', '', '', '', '', '', ''];
+
+function endGameColor() {
```

Comment:
[OPTIONAL]
Since you are using ES6 syntax in other parts, consider using it here as well